docs/ui/cloud-concepts.md

```diff
@@ -0,0 +1,194 @@
+1. [Run a flow from a deployment](#run-a-flow-from-a-deployment), enabling remote execution of flow runs.
+
+## Sign in or register
+
+To sign in with an existing account or register an account, go to [https://app.prefect.cloud/](https://app.prefect.cloud/).
+
+You can create an account with:
+
+- Google account
+- Microsoft (GitHub) account
+- Email and password
+
+## Create a workspace
+
+A workspace is an isolated environment within Prefect Cloud for your flows and deployments. You can use workspaces to organize or compartmentalize your workflows. For example, you can create separate workspaces to isolate development, staging, and production environments; or to provide separation between different teams.
+
+When you register a new account, you'll be prompted to create a workspace.  
+
+![Creating a new Prefect Cloud account.](/img/ui/cloud-new-login.png)
+
+Select **Create Workspace**. You'll be prompted to provide a name and description for your workspace.
+
+![Creating a new workspace in the Cloud UI.](/img/ui/cloud-workspace-details.png)
+
+Note that the **Owner** setting applies only to users who are members of Prefect Cloud organizations who have permission to create workspaces within the organization.
+
+Select **Save** to create the workspace. If you change your mind, select **Edit** from the options menu to modify the workspace details or to delete it. 
+
+![Viewing a workspace dashboard in the Prefect Cloud UI.](/img/ui/cloud-new-workspace.png)
+
+The **Workspace Settings** page for your new workspace displays the commands to install Prefect and log into Prefect Cloud in a local execution environment.
+
+## Log into Prefect Cloud from a terminal
+
+Configure a local execution environment to use Prefect Cloud as the API server for flow runs. In other words, "log in" to Prefect Cloud from a local environment where you want to run a flow.
+
+1. Open a new terminal session.
+2. [Install Prefect](/getting-started/installation/) in the environment in which you want to execute flow runs.
+
+<div class="terminal">
+```bash
+$ pip install -U prefect
+```
+</div>
+
+3. Use the `prefect cloud login` Prefect CLI command to log into Prefect Cloud from your environment.
+
+<div class="terminal">
+```bash
+$ prefect cloud login
+```
+</div>
+
+The `prefect cloud login` command, used on its own, provides an interactive login experience. Using this command, you may log in with either an API key or through a browser.
+
+<div class="terminal">
+```bash
+$ prefect cloud login
+? How would you like to authenticate? [Use arrows to move; enter to select]
+> Log in with a web browser
+    Paste an authentication key
+Paste your authentication key:
+? Which workspace would you like to use? [Use arrows to move; enter to select]
+> prefect/terry-prefect-workspace
+    g-gadflow/g-workspace
+Authenticated with Prefect Cloud! Using workspace 'prefect/terry-prefect-workspace'.
+```
+</div>
+
```

Comment:
Maybe add:
"Alternatively, log in by copy/pasting an API key." With a link to how to create an API key or or a screenshot showing where to grab an API.